cf_remote/packages.py

```diff
@@ -130,17 +139,25 @@ def filter_artifacts(artifacts, tags, extension):
 
 
 class Release:
-    def __init__(self, data):
-        self.data = data
-        self.version = data["version"]
-        self.url = data["URL"]
-        self.lts = data["lts_branch"] if "lts_branch" in data else None
-        self.extended_data = None
+    def __init__(self, data, extended_data=None):
+        if extended_data:
+            self.data = extended_data
+            self.extended_data = extended_data
+            self.url = "no-url"
```

Comment:
This block looks weird. `extended_data` is assigned to **both** `self.data` and `self.extended_data` with the latter being thrown away at the first `init_download()` call. And what does it have to do with arm64 support?

Reply:
I could re-work that as a single commit related to the test I wrote to support refactoring to support arm64.

Reply:
Please do.

Reply:
Yeah, I realize looking at it now that it is more functional than readable. I'll make two commits and maybe add a few more tests. :+1:

Reply:
Calling the parameter `external_data` might also help.